src/main/java/com/stuypulse/robot/subsystems/Shooter.java

```diff
@@ -90,6 +90,10 @@ public Shooter() {
         hoodSolenoid = new Solenoid(PneumaticsModuleType.CTREPCM, Ports.Shooter.HOOD_SOLENOID);
     }
 
+    public void setShooterRPM(SmartNumber speed) {
+        targetRPM.set(speed);
+    }
+
```

Comment:
see comment above about this method. 
for more clarity, smart number extends Number (thanks to sam)
double will be converted into Number (thanks to java auto-boxing)

so only one method that takes in a number is necessary. this fits nicely with storing a NUmber in the command (like i mentioned above)

the way you have it now works but i figured i'd mention it because the number polymorphism is kinda cool and shouldnt cause you to make more code.